logforwarder/src/apps/vespa-logforwarder-start/child-handler.cpp

```diff
@@ -33,10 +39,10 @@ runSplunk(const vespalib::string &prefix, const char *a1, const char *a2 = 0)
         char *cenv = const_cast<char *>(env.c_str()); // safe cast
         putenv(cenv);
         LOG(debug, "added to environment: '%s'", cenv);
-        char **cargv = const_cast<char **>(argv); // safe cast
-        execv(argv[0], cargv);
+        char **cargv = const_cast<char **>(args.data()); // safe cast
```

Comment:
Missing final `nullptr` entry in `args` vector to signal end of argument list?

Reply:
> Missing final nullptr entry in args vector to signal end of argument list?

yes, thank you!